### ISSUE TITLE
Publish braviarc on pypi

### ISSUE THREAD

**Issue by andrey-git** [aparraga/braviarc#12]
Could you publish this package on Pypi, so it can be fetched without specifying the github URL?

home-assistant/home-assistant#7069

Thanks!

**Comment by gotsanity** [comment]
I agree, I am using this project in a kivy android app for a class and it would make including it in the APK much easier.

**Comment by Knapoc** [comment]
@aparraga is there any reason why braviarc is not hosted on pypi?
